app/controllers/spectrums_controller.rb

```diff
@@ -238,7 +238,7 @@ def create
               #@spectrum.tag("mobile", user.id)
             end
  
-            if params[:spectrum][:json].nil? && params[:vertical] == "on"
+            if params[:spectrum][:json]=="" && params[:vertical]
```

Comment:
Maybe adding some spaces here would be nice :) 
``` ruby
params[:spectrum][:json] == ""
```

Reply:
Done!